lib/ui/account_details_form/account_details_form.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import 'package:sunshine/core/core.dart';
+import 'package:sunshine/sunshine.dart';
+import 'account_details_form_view_model.dart';
+
+class AccountDetailsForm extends StatelessWidget {
+  AccountDetailsForm({
+    Key key,
+    this.formKey,
+    this.autoValidate = false,
+  }) : super(key: key);
+
+  final GlobalKey<FormState> formKey;
+  final bool autoValidate;
+  final _firstNameNode = FocusNode();
+  final _lastNameNode = FocusNode();
+  final _passwordNode = FocusNode();
+  final _confirmPasswordNode = FocusNode();
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
```

Comment:
there is a small issue here I should fix first, it only happens on very small screens, the last input (confirm password) is hidden (below the keyboard)